File: src/main/java/dao/DatabaseOperate.java
package dao;

import pojo.DeskChange;
import pojo.StaffPersonal;
import pojo.StaffWork;

/**
 * @author Kayll
 * @date 2018/11/3 18:42
 */
public interface DatabaseOperate {
    public boolean loginForRegist(String account, String password);
    public boolean loginForMain(String account, String password);
    public StaffWork selectStaffWork(String email);
    public int updateStaffPersonal(StaffPersonal staffPersonal);
    public int insertDeskChange(DeskChange deskChange);
}

File: src/main/java/dao/DatabaseOperate.kt
package dao

import pojo.DeskChange
import pojo.StaffPersonal
import pojo.StaffWork

/**
 * @author Kayll
 * @date 2018/11/3 18:42
 */
interface DatabaseOperate {
    fun loginForRegist(account: String, password: String): Boolean
    fun loginForMain(account: String, password: String): Boolean
    fun selectStaffWork(email: String): StaffWork?
    fun updateStaffPersonal(staffPersonal: StaffPersonal): Int
    fun insertStaffWork(staffWork: StaffWork): Int
    fun updateStaffPersonalWorkId(): Int
    fun insertDeskChange(deskChange: DeskChange): Int
}
